EIPS/eip-6865.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 6865
+title: on-chain EIP-712 Visualization
+description: on-chain visualization of EIP-712 payloads into structured data that highlights the potential consequences for users' assets
+author: Abderrahmen Hanafi (@a6-dou)
+discussions-to: https://ethereum-magicians.org/
```

Comment:
please create an actual post on `ethereum-magicians` and use as your discussions to link

Reply:
Thanks for the review, [Pushed the update](https://github.com/ethereum/EIPs/pull/6865/commits/46fb7a3884be877e52601359e4a56add54372c48) with discussion link